apps/dashboard/src/lib/components/Org/Settings/index.svelte

```diff
@@ -45,35 +45,35 @@
     tabs = [
       {
         key: 0,
-        label: $t('settings.tabs.1'),
+        label: $t('settings.tabs.profile_tab'),
         tabKey: '',
         href: $page.url.pathname,
         disabled: false
       },
       {
         key: 1,
```

Comment:
This PR looks like a mix with some translation fixes.

Please pull the latest of main into this branch